File: app/src/main/java/onefootball/com/testonefootball/data/PushRepositoryImpl.java
package onefootball.com.testonefootball.data;

import java.util.HashMap;

public class PushRepositoryImpl implements PushRepository
{
    final HashMap<Long, PushItem> pushMap = new HashMap<>();

    @Override
    public void setTeamPush(long teamId, String name, int pushOptions)
    {
        PushItem item = new PushItem();
        item.setPushItemType(PushItem.TYPE_TEAM_PUSH);
        item.setPushItemId(teamId);
        item.setPushItemName(name);
        item.setPushOption(String.valueOf(pushOptions));

        pushMap.put(teamId, item);
    }

    @Override
    public PushItem getTeamPush(long teamId)
    {
        return pushMap.get(teamId);
    }
}

File: app/src/main/java/onefootball/com/testonefootball/data/PushRepositoryImpl.kt
package onefootball.com.testonefootball.data

import java.util.HashMap

import onefootball.com.testonefootball.data.model.PushItem

class PushRepositoryImpl : PushRepository {
    private val pushMap: HashMap<Long, PushItem> = hashMapOf()

    override fun setTeamPush(teamId: Long, name: String, pushOptions: Int) {
        val item = PushItem(teamId, PushItem.TYPE_TEAM_PUSH, name, pushOptions.toString())
        pushMap[teamId] = item
    }

    override fun getTeamPush(teamId: Long): PushItem? {
        return pushMap[teamId]
    }
}
